README.md

```diff
@@ -78,9 +82,18 @@
 
  ‡ A local PostgreSQL server installation is not required. `cargo pgrx` can download and compile PostgreSQL versions on its own.
 
+ ⹋ PGRX has not been tested to work on 32-bit: the library assumes an 8-byte `pg_sys::Datum`
+which may result in unexpected behavior on 32-bit, like dropping 4 bytes of data from `int8`
+and `double`. This is not "unsound", but may undermine safety assumptions of PGRX extensions.
```

Comment:
I don't like the overstated confidence here:

```suggestion
and `double`. This may not be "unsound" in itself, as it is "merely" illogical,
but it may undermine otherwise-reasonable safety assumptions of PGRX extensions.
```